validator/src/main/java/org/odftoolkit/odfvalidator/Main.java

```diff
@@ -213,7 +214,7 @@ public static void main(String[] aArgs) {
         aConfig.setProperty(Configuration.DSIG_SCHEMA, aDSigSchemaFileName);
       }
       if (aMathMLSchemaFileName != null) {
-        aConfig.setProperty(Configuration.MATHML2_SCHEMA, aMathMLSchemaFileName);
+        aConfig.setProperty(Configuration.MATHML3_SCHEMA, aMathMLSchemaFileName);
```

Comment:
this breaks the build, but it's fixed in the next commit

Reply:
as I said, I split the 3 commits after fixing alls and very roughly.. ;-)